docs/cookbook/events_and_gestures.md

```diff
@@ -107,6 +107,19 @@ end
 
 ```
 
+## Custom events
+
+To add a custom event, use `.on` with a custom symbol. Call `.trigger` to trigger your block.
+
+```ruby
+
+# this code allows you to add a custom event and trigger it
+rmq.append(UIView, :my_view).on(:custom_event) do |sender|
+    puts "custom_event has been triggered"
+end
+rmq(:my_view).trigger(:custom_event)
```

Comment:
since we're documenting this in redpotion we can change these up a bit to match the redpotion idioms.

```ruby
append(UIView, :my_view).on(:custom_event) do |sender|
  puts "custom_event has been triggered"
end

find(:my_view).trigger(:custom_event)
```

other than that - looks good to me - thanks for adding the feature **and** the docs!